.github/workflows/ci.yml

```diff
@@ -5,51 +5,98 @@ defaults:
   run:
     working-directory: app
 
-jobs:  
-  ci:
-    name: Run eslint, stylelint and tests
-    runs-on: ubuntu-18.04
+jobs:
+  linting:
+    name: eslint, stylelint
+    runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: [ 14.x ]
+        node-version: [14.x]
     steps:
+      # common steps
       - uses: actions/checkout@v2
-        with:
-          # Chromatic needs full history
-          fetch-depth: 0
       - name: Use Node.js ${{ matrix.node-version }}
-        uses: actions/setup-node@v1
+        uses: actions/setup-node@v2
         with:
           node-version: ${{ matrix.node-version }}
       - name: Install dependencies
         run: |
           npm config set //registry.npmjs.org/:_authToken=$NPM_TOKEN
-          npm i -g npm@lts
+          npm install -g npm@lts
           npm ci
-          npm rebuild node-sass
+      # job specific steps
       - name: Run ESlint
         run: npm run eslint
       - name: Run Stylelint
         run: npm run stylelint
-      - name: Build storybook
-        run: npm run build-storybook --output-dir storybook-static
-      - name: Publish storybook to Chromatic
-        uses: chromaui/action@v1
+
+  jest:
+    name: Jest Tests
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [14.x]
+    steps:
+      # common steps
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v2
         with:
-          token: ${{ secrets.GITHUB_TOKEN }}
-          projectToken: ${{ secrets.CHROMATIC_PROJECT_TOKEN }}
-          exitZeroOnChanges: true
-          workingDir: app
-          # Use output folder from previous step
-          storybookBuildDir: storybook-static
+          node-version: ${{ matrix.node-version }}
+      - name: Install dependencies
+        run: |
+          npm config set //registry.npmjs.org/:_authToken=$NPM_TOKEN
+          npm install -g npm@lts
+          npm ci
+      # job specific steps
       - name: Build app
-        run: npm run build --if-present
+        run: npm run build
       - name: Start Streamr Docker Stack
-        uses: streamr-dev/streamr-docker-dev-action@v1.0.0-alpha.2
+        uses: streamr-dev/streamr-docker-dev-action@v1.0.0-alpha.3
```

Comment:
`v1.0.0-alpha.2` had problems with `ubuntu-latest` but great that this fixes it 👍